third_party/nvidia/backend/compiler.py

```diff
@@ -234,7 +234,9 @@ def make_ttgir(mod, metadata, opt, capability):
         passes.ttgpuir.add_remove_layout_conversions(pm)
         passes.ttgpuir.add_optimize_thread_locality(pm)
         passes.ttgpuir.add_accelerate_matmul(pm)
+        passes.ttgpuir.add_optimize_gather_layouts(pm)
         passes.ttgpuir.add_remove_layout_conversions(pm)
+        passes.common.add_licm(pm)
```

Comment:
I remembered that there are some convert layouts we don't always want to move out of the loop. Can you run a perf regression test?

Reply:
How do I run the regression tests? Also, I'm not certain this is necessary anymore. I'll investigate removing it.

Reply:
cc @pawelszczerbuk or @peterbell10  can you help Jeff with running the perf regression tests?

Reply:
@lezcano Sent me the instructions. Thanks! I reran my benchmarks for gather, and this isn't needed anymore anyways.